File: atcoder/373/B.py
def main():
    S = input()
    X = [0] * 26
    for i in range(26):
        X[ord(S[i]) - ord('A')] = i
    ans = 0
    for i in range(25):
        sub = abs(X[i] - X[i + 1])
        ans += sub
    print(ans)
    
if __name__ == "__main__":
    main()

File: atcoder/373/B.cpp
#include <iostream>
#include <string>
#include <cmath>
#include <vector>
using namespace std;

int main()
{
    vector<int> X(26);
    string S;
    cin >> S;
    for (int i = 0; i < 26; ++i)
    {
        X[S[i] - 'A'] = i;
    }
    int sum = 0;
    for (int i = 0; i < 25; i++)
    {
        int sub = abs(X[i] - X[i + 1]);
        sum += sub;
    }
    cout << sum << endl;
    return 0;
}